src/app/operations/operation-path.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-operation-path',
+  template: `
+    <pre class="definition"><code><span class="http_method {{methodType}}">{{methodType}}</span><span class="path">{{path}}</span></code></pre>
+  `,
+  styles: []
+})
+export class OperationPathComponent implements OnInit {
```

Comment:
That's just the default structure from the ng-cli generator. I'm not against removing those.